### PR TITLE
Update policy-csp-authentication.md

### PR DESCRIPTION
**Pull request by mapalko**
Update to the policy definition for secondary authentication factor. The default behavior for consumer devices will change in the next major WIndows 10 release.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by nenonix** [comment]
@mapalko please could you reference the issue in your commit notes by pasting in the URL?

File: windows/client-management/mdm/policy-csp-authentication.md [hunk 1]
```diff
@@ -296,6 +296,8 @@ Added in Windows 10, version 1607. Allows secondary authentication devices to w
 
 The default for this policy must be on for consumer devices (defined as local or Microsoft account connected device) and off for enterprise devices (such as cloud domain-joined, cloud domain-joined in an on-premises only environment, cloud domain-joined in a hybrid environment, and BYOD).
 
+In the next major release of Windows 10, the default for this policy for consumers devices will be changed to off. This will only affect users that have not already set up a secondary authentication device.
```
Comment (nenonix) [inline windows/client-management/mdm/policy-csp-authentication.md]: Suggestions
1) "the next major release of Windows 10" could be given a date or a name or something to locate change in a timeline and keep this information in context in the future.
2) the default for this policy **will be changed to off for consumer devices**. This change will only affect users **who have not yet** set up a secondary authentication device.

**Comment by jdeckerms** [comment]
verifying source of the information before merging

**Pull request merged**